server/src/main/java/org/spine3/server/stand/SubscriptionRegistry.java

```diff
@@ -68,7 +69,8 @@ synchronized void activate(Subscription subscription, Stand.EntityUpdateCallback
     synchronized Subscription addSubscription(Target target) {
         final String subscriptionId = Identifiers.newUuid();
         final String typeAsString = target.getType();
-        final TypeUrl type = TypeUrl.of(typeAsString);
+        final TypeUrl type = TypeName.of(typeAsString)
+                                     .toUrl();
```

Comment:
We had this ambiguity before. A type name is not a type URL. Here we convert to a URL via internal knowledge of all known types.